requirements.txt

```diff
@@ -2,5 +2,6 @@ globus-sdk>=3.0
 httpx>=0.22.0
 python-dotenv
 prefect==2.7.9
+pydantic<2.0
```

Comment:
Is this just to lock in dependencies from other packages?

Reply:
Oh, I should explained. This old version of prefect forgot to pin pydnatic. So if you build a new enviornment today, you get 2.x version of pydantic, which breaks prefect.

Reply:
Also, I looked briefly into upgrading prefect because of this, but they refactored the API enough that I can't do that now.